client/lib/user/support-user-interop.js

```diff
@@ -132,20 +144,22 @@ export const boot = () => {
 		return;
 	}
 
-	localforage.bypass();
-
-	const { user, token } = store.get( STORAGE_KEY );
+	const { user, token } = JSON.parse( window.sessionStorage.getItem( STORAGE_KEY ) );
```

Comment:
Let's also add a try catch here, with an early return in the catch.

Reply:
updated on 04e33a972fe2a8530051d8d2c12b13850acca81b.